cmake/toolschains/ubuntu1804-mingw64.cmake

```diff
@@ -0,0 +1,31 @@
+# Sample toolchain file for building for cross-compiling to Windows from an Ubuntu Linux system.
+#
+# Typical usage:
+#    *) install cross compiler: `sudo apt install mingw-w64`
+#    *) mkdir buildMingw64 && cd buildMingw64
+#    *) cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/ubuntu1804-mingw64.cmake ..
+#
+
+set(CMAKE_SYSTEM_NAME Windows)
+set(TOOLCHAIN_PREFIX x86_64-w64-mingw32)
+
+# cross compilers to use for C and C++
+# Note: I had to use the posix variant to avoid compilation errors related with not finding the mutex class
+set(CMAKE_C_COMPILER ${TOOLCHAIN_PREFIX}-gcc-posix)
+set(CMAKE_CXX_COMPILER ${TOOLCHAIN_PREFIX}-g++-posix)
+set(CMAKE_RC_COMPILER ${TOOLCHAIN_PREFIX}-windres)
+
+# target environment on the build host system
+set(CMAKE_FIND_ROOT_PATH /usr/${TOOLCHAIN_PREFIX} /usr/lib/gcc/${TOOLCHAIN_PREFIX}/7.3-posix)
```

Comment:
Question: in README-crosscompiling you mentioned that an error can appear when the mingw-win32 compiler is used. But here posix is hardcoded, so that issue shouldn't be possible, right? (provided people actually use our toolchain file)

Reply:
Exactly, this toolchain should always work on Ubuntu 18.04. The comment in the file `README-crosscompiling.md` is a general statement that could be helpful for people trying to write toolchain files for other linux distributions or Ubuntu versions.